packages/headless/src/api/search/search/search-response.ts

```diff
@@ -8,13 +8,15 @@ import {AnyFacetResponse} from '../../../features/facets/generic/interfaces/gene
 import {QueryRankingExpression} from './query-ranking-expression';
 import {SecurityIdentity} from './security-identity';
 import {ExecutionReport} from './execution-report';
+import {QuestionsAndAnswers} from './question-answering';
 
 export interface SearchResponseSuccess {
   results: Result[];
   searchUid: string;
   totalCountFiltered: number;
   facets: AnyFacetResponse[];
   queryCorrections: QueryCorrection[];
+  questionAnswer?: QuestionsAndAnswers;
```

Comment:
The way it works from the API standpoint: 

If there's a question + answer model configured, this is populated.

Otherwise, it's not sent.

I could "cheat" here, set it as non optional, and populate it with an "empty object" in search slice when it's not available, so that for the rest of the code there's a bit less null check everywhere.

What do you prefer ?

Reply:
Then all the QuestionsAndAnswers params would be optional?